modules/ingestor/src/graph/sbom/common/package.rs

```diff
@@ -109,9 +115,44 @@ impl PackageCreator {
 
         self.packages.push(sbom_package::ActiveModel {
             sbom_id: Set(self.sbom_id),
-            node_id: Set(node_id),
+            node_id: Set(node_id.clone()),
             version: Set(version),
         });
+
+        if let Some(declared) = declared_licenses {
+            self.sbom_packge_licenses
+                .push(sbom_package_license::ActiveModel {
+                    id: Default::default(),
+                    sbom_id: Set(self.sbom_id),
+                    node_id: Set(node_id.clone()),
+                    license_id: Set(declared.uuid()),
+                    license_type: Set(sbom_package_license::LicenseCategory::SpdxDeclared),
+                });
+        }
+
+        if let Some(concluded) = concluded_licenses {
+            self.sbom_packge_licenses
+                .push(sbom_package_license::ActiveModel {
+                    id: Default::default(),
+                    sbom_id: Set(self.sbom_id),
+                    node_id: Set(node_id.clone()),
+                    license_id: Set(concluded.uuid()),
+                    license_type: Set(sbom_package_license::LicenseCategory::SpdxConcluded),
+                });
+        }
+
+        if let Some(cyclonedx) = cyclonedx_licenses {
+            for (uuid, _v) in cyclonedx.licenses {
+                self.sbom_packge_licenses
+                    .push(sbom_package_license::ActiveModel {
+                        id: Default::default(),
+                        sbom_id: Set(self.sbom_id),
+                        node_id: Set(node_id.clone()),
+                        license_id: Set(uuid),
+                        license_type: Set(sbom_package_license::LicenseCategory::Other),
```

Comment:
Why does CDX become "other". To my understanding CDX has declared and concluded as well. Which also means that it doesn't make sense to call the other variants "Spdx*"

Reply:
The acknowledgemen[1] have types declared and concluded , but I have never seen any CycloneDX having this property. Therefore, we do not know which type of acknowledgement the CycloneDX license belongs to.

[1] https://cyclonedx.org/docs/1.6/json/#components_items_licenses_oneOf_i0_items_license_acknowledgement

Reply:
This relates to https://github.com/trustification/trustify/pull/1331#discussion_r1969547113